docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_updateManagementCenter/_index.md

```diff
@@ -0,0 +1,180 @@
+
+    ![Missing updates](./11.png)
+
+- If you click on the **History** tab, you will see the details related to the updates assessment and operations:
+
+    ![Update assessment history](./12.png)
+
+## Update Deployment
+
+On the other hand, as part of this scenario, a new **Maintenance Configuration** resource is created. It allows you to schedule recurring updates deployments on your Azure Arc-enabled servers.
+
+- Go to your **resource group** and click on the new **Maintenance Configuration** resource:
+
+    ![New maintenance configuration resource](./13.png)
+
+- Review the following **Maintenance Configuration** settings: **Machines**, which includes the Azure Arc-enabled server; **Schedule** of the maintenance configuration resource (it will start one day after you deploy this scenario); and **Updates** settings:
+
+    ![Maintenance configuration resource - machines](./14.png)
+
+    ![Maintenance configuration resource - schedule](./15.png)
+
+    ![Maintenance configuration resource - updates](./16.png)
+
+- Once the **Schedule** triggers the **Maintenance Configuration**, you will be able to see the results. Go back to your **Azure Arc-enabled server**, click on **Updates**, and then on **History**:
+
+    ![Update deployment history](./17.png)
+
+    ![Update deployment history details](./18.png)
+
+## Update Management Center
+
+From **Update Management Center**, you can oversee the update compliance for your entire fleet of machines in Azure, on-premises, and other cloud environments.
+Moreover, you can instantly deploy critical updates to help secure your machines.
+
+- On the top bar search, type **Update Management Center** and click on it:
+
+    ![Update Management Center](./19.png)
+
+- The **Overview** blade shows you the current updates status of your hybrid environment:
+
+    ![Update Management Center - Overview](./20.png)
+
+- If you click on **Machines**, you will get more details per each individual machine:
+
+    ![Update Management Center - Machines](./21.png)
+
+- Moreover, **History** shows you all the updates activities on your hybrid environment:
+
+    ![Update Management Center - History](./22.png)
+
+- Lastly, under the **Workbooks** section, there is a built-in workbook that provides an unified view for your current updates status:
+
+    ![Update Management Center - Workbooks](./23.png)
+
+## Clean up environment
+
+Complete the following steps to clean up your environment:
+
+- [Delete a maintenance configuration](https://learn.microsoft.com/azure/virtual-machines/maintenance-configurations-cli#delete-a-maintenance-configuration)
+- [Remove extension on Azure Arc-enabled server](https://learn.microsoft.com/azure/azure-arc/servers/manage-vm-extensions-cli#remove-extensions)
+- [Delete Azure Policy Assignment](https://learn.microsoft.com/cli/azure/policy/assignment?view=azure-cli-latest#az-policy-assignment-delete)
```

Comment:
We should show how to do it so the reader won't have to figure this out. Let's do it "Jumpstart style" :-)